---
identifier: "CVE-2020-26244"
package_slug: "pypi/oic"
title: "Improper Verification of Cryptographic Signature"
description: "Python oic is a Python OpenID Connect implementation. In Python oic,
  there are several related cryptographic issues affecting client implementations
  that use the library."
date: "2020-12-08"
pubdate: "2020-12-02"
affected_range: "<1.2.1"
fixed_versions:
- "1.2.1"
affected_versions: "All versions before 1.2.1"
not_impacted: "All versions starting from 1.2.1"
solution: "Upgrade to version 1.2.1 or above."
urls:
- "https://nvd.nist.gov/vuln/detail/CVE-2020-26244"
- "https://pypi.org/project/oic/"
cvss_v2: "AV:N/AC:M/Au:S/C:P/I:P/A:N"
cvss_v3: "CVSS:3.1/AV:N/AC:H/PR:L/UI:N/S:U/C:H/I:H/A:N"
uuid: "449091b8-538d-438f-87b6-cc21904b6591"
cwe_ids:
- "CWE-1035"
- "CWE-347"
- "CWE-937"
identifiers:
- "CVE-2020-26244"
- "GHSA-4fjv-pmhg-3rfg"
